sample/build.gradle

```diff
@@ -50,6 +50,7 @@ android {
             signingConfig signingConfigs.release
 		}
 	}
+    dataBinding.enabled = true
```

Comment:
"Data Binding Library to write declarative layouts and minimize the glue code necessary to bind your application logic and layouts." from Android doc